### ISSUE TITLE
deb file upload error

### ISSUE THREAD

**Issue by myururdurmaz** [go-gitea/gitea#26022]
### Description

Hi!
I  can't upload my deb package gitea package registry. Curl request returning "unsupported compression algorithm". Because file names in my deb file ends with slash. 
```
head -100c yonca-scripts_0.0.168_amd64.deb 
!<arch>
debian-binary/  0           0     0     644     4         `
2.0
control.tar.xz/ 0   
```        
I found bellow message on this url https://man7.org/linux/man-pages/man5/deb-split.5.html

       The file is an ar archive with a magic value of !<arch>.  The
       file names might contain a trailing slash (since dpkg 1.15.6). 

### Gitea Version

1.20.0

### Can you reproduce the bug on the Gitea demo site?

Yes

### Operating System

debian linux

### How are you running Gitea?

systemd service on linux server

### Database

PostgreSQL

**Comment by KN4CK3R** [comment]
Could you share the deb file so I can try?

**Comment by myururdurmaz** [comment]
here a test deb file

[deneme_1_all.deb.gz](https://github.com/go-gitea/gitea/files/12123523/deneme_1_all.deb.gz)

**Comment by KN4CK3R** [comment]
Thank you for the file. Fixed in #26087.

**Comment by myururdurmaz** [comment]
thanks
